lenskit-core/src/main/java/org/lenskit/hybrid/Left.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * LensKit, an open source recommender systems toolkit.
+ * Copyright 2010-2014 LensKit Contributors.  See CONTRIBUTORS.md.
+ * Work on LensKit has been funded by the National Science Foundation under
+ * grants IIS 05-34939, 08-08692, 08-12148, and 10-17697.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package org.lenskit.hybrid;
+
+import org.grouplens.grapht.annotation.DefaultImplementation;
+import org.lenskit.basic.TopNItemRecommender;
+
+import javax.inject.Qualifier;
+import java.lang.annotation.*;
+
+/**
+ * The 'left' recommender for a hybrid.
+ */
+@Qualifier
+@Documented
+@DefaultImplementation(TopNItemRecommender.class)
```

Comment:
Generally I feel that this is part of the tensions between reusing annotations (good in some places) and having annotations be specific to one class (good in other places). Is it possible to have an annotation be a subclass of another class so it would be `@RankBlendingItemScorer.Left`? That would avoid the concern in this case where "left" is too generic to be a reused annotation